### ISSUE TITLE
How to Customizes generation JsonSchema like newtonsoft JSchemaGenerationProvider?

### ISSUE THREAD

**Issue by xiaguoli** [RicoSuter/NJsonSchema#1212]
First of all, thank you for the library!

I want to assign a value  to every Schema.ExtensionData according to its attribute  ,But I don't know how to do it. use the SchemaProcessor?

**Comment by RicoSuter** [comment]
Yes, have you tried a schema processor? I think it's called for every schema.
